code/js/modules/BaseController.js

```diff
@@ -133,7 +159,18 @@
   };
 
   BaseController.prototype.mute = function() {
-    this.click({action: "mute", selectorButton: this.selectors.mute, selectorFrame: this.selectors.iframe});
+    // TODO: volume of video tab and website state is different
```

Comment:
What do you mean by this?

Reply:
I am not sure about this, at an earlier commit it said 

```javascript
// TODO: volume of media tab and website state is different
```